### DISCUSSION TITLE
Practica pastelería

### DISCUSSION THREAD

**Discussion by Rodrigorenteria** [Launch-X-Latam/MisionFrontEnd#319] [Show and tell]
Hola! así quedó mi página de la pastelería jeje, espero les guste y me puedan dar retroalimentación.
Pd: le invertí bastante tiempo y vi muchos tutoriales en internet jajaja

[Pastelería](https://rodrigorenteria.github.io/Pasteleria/)

**Comment by AlvaroFierro** [comment]
Te quedo muy bien 👀
En verdad se nota el esfuerzo que le has puesto
Sin embargo, buscaria la manera de hacer menos pesadas las imagenes, porque si es bastante y tarda algo en cargar la pagina, pero seria lo unico

**Reply by Rodrigorenteria** [reply]
Muchas gracias! La verdad no pensé en el peso de los archivos pero si voy a comprimir las imágenes para que cargue más rápido 😊

**Reply by AlvaroFierro** [reply]
No hay de que, para eso es la retro

**Comment by ghost** [comment]
Quedo muy bien, yo no tuve problema en cargar tu página, carga rápido.

**Reply by AlvaroFierro** [reply]
Igual a mi me cargo rapido, pero cuando te vas a las opciones de desarrollador para inspeccionar, son > 20 MB que tiene que descargar el usuario y recordemos que ahora el mayor trafico proviene desde un celular. Y no todos tienen muchos datos. Mas que nada es solo un comentario para que lo tengan en cuenta. De hecho el creador de contenido midudev en uno de sus lives hizo notar esa parte 

Les recomiendo leer sobre las imagenes `.webp` y sobre `lazy loading` 

Foto de las opciones de desarrollador 
![image](https://user-images.githubusercontent.com/71525061/200135807-39c99298-9406-4ffa-a8d8-35e5c51b19e4.png)
